src/app/progress/milestone.scss

```diff
@@ -146,6 +146,40 @@
   }
 }
 
-.well-rested {
-  cursor: none;
+.well-rested,
+.seasonal-rank {
+  cursor: default;
+}
+
+.seasonal-rank {
+  .milestone-icon {
+    max-width: calc(var(--item-size) * 1.2);
+  }
+
+  &.has-premium-rewards {
+    .milestone-icon {
+      max-width: calc(var(--item-size) * 2.2);
+      flex-direction: row;
+      flex-wrap: wrap;
+    }
+  }
+
+  .seasonal-reward-wrapper {
+    border: 2px solid #fff;
```

Comment:
For items we use a 1px border (actually `$item-border-width` in SCSS). Since some of these are transparent we should probably also give it a background color.

Check out PursuitImage for some styles that will match up with our existing pursuits display.